docs/data/destinations/blitzllama-cohort.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Send Cohorts to Blitzllama
+description: Send Amplitude cohorts to Blitzllama.
+---
+
+--8<-- "includes/editions-all-paid-editions.md"
+
+[Blitzllama](https://www.linkedin.com/company/blitzllama/) GPT-powered feedback analytics automatically categorizes survey responses into topics, and sentiments, and provides recommendations. This saves time that would be spent manually summarizing the feedback. As all the feedback is stored in one place, you can segment historical feedback and create charts within a few clicks.
```

Comment:
@Brandoncharleskhoo1 Any reason we're linking to LInkedIn here, and not their website?